src/utils/Printer.cpp

```diff
@@ -0,0 +1,72 @@
+//
+// Created by DANIELA CARRERO on 2020-10-25.
+//
+
+#include <SDL_ttf.h>
```

Comment:
Please use 
#include <SDL2/SDL_ttf.h>
(If not working with the rest of your code then put an if APPLE if you wish)

Reply:
Listo!